var/spack/repos/builtin/packages/n2p2/package.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class N2p2(MakefilePackage):
+    """N2p2 (The neural network potential package) provides ready-to-use
+       software for high-dimensional neural network potentials
+       in computational physics and chemistry."""
+
+    homepage = "https://github.com/CompPhysVienna/n2p2"
+    url = "https://github.com/CompPhysVienna/n2p2/archive/v2.1.0.tar.gz"
+
+    version(
+        "2.1.1",
+        sha256="90fbc0756132984d0d7e6d92d2f53358c120e75f148910d90c027158163251b9",
+    )
+    version(
+        "2.1.0",
+        sha256="283c00e9a5b964f4c84a70c5f1cef7167e9b881080b50a221da08799e5ede400",
+    )
+
+    variant("doc", default=False, description="build documentation with Doxygen")
+
+    patch("interface-makefile.patch", when="@2.1.0")
+    patch("interface-makefile211.patch", when="@2.1.1")
+    patch("libnnp-makefile.patch")
+    patch("nnp_test.h.patch")
+
+    depends_on("mpi")
+    depends_on("blas")
+    depends_on("gsl", type=("build", "run", "link"))
+    depends_on("eigen", type=("build", "run", "link"))
+    depends_on("doxygen", type="build", when="+doc")
+    depends_on("texlive", type="build", when="+doc")
+    depends_on("py-cython", type=("build", "run"))
```

Comment:
This is usually build-only

Reply:
I tried it. But I got a error of cypthon in spack test.
The error was "ModuleNotFoundError: No module named 'Cython'".
So I can not change type=("build", "run") to type="build"